components/director/pkg/graphql/schema.graphql

```diff
@@ -51,6 +51,15 @@ enum ApplicationStatusCondition {
 	INITIAL
 	CONNECTED
 	FAILED
+	CREATING
```

Comment:
You need to add these types to the model + db layers as well. This includes migrations to the enum field.
Also I noticed that after you do that, you need to adapt the converters in places like this one: https://github.com/kyma-incubator/compass/blob/16d72a407f83a06550044ca11ad07fd6050cfe25/components/director/internal/domain/application/converter.go#L211